package.json

```diff
@@ -57,8 +57,15 @@
     "yarn-run-all": "^3.1.1"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.36",
```

Comment:
put the packages in the package.json you are actually using them

Reply:
I'm actually not using them, I use react-icons. I needed my work to run so I just installed them when I noticed it was missing

Reply:
check which spa uses it so you can move them into that